react-router/index.ts

```diff
@@ -0,0 +1,29 @@
+import { match, matchPath } from 'react-router';
+import { RouterState } from '../src/reducer';
+
+interface Options {
+   selectRouterState?: <S>(state: S) => RouterState;
+   routerReducerKey?: string;
+}
+
+export const createMatchSelector = (path: string, options?: Options) => {
```

Comment:
The signature should also allow an array of strings (as the `connected-react-router` signature does)

```suggestion
export const createMatchSelector = (path: string | string[], options?: Options) => {
```